workspaces/editors.md

```diff
@@ -149,15 +149,6 @@ The symlink names supported by Coder are:
 - `studio` (Android Studio)
 - `webstorm`
 
-### System requirements
-
-We recommend running the JetBrains IDE in an workspace with the following
-minimum resources but the resources required depends on your workspace-specific
-requirements:
```

Comment:
Should we direct users to consult the documentation for their favoured IDE? 
For Idea, we could link directly to https://www.jetbrains.com/help/idea/installation-guide.html#requirements

Reply:
How would we do this efficiently without linking to each IDE's specific requirements?

Reply:
I don't think there's a great way of linking to all the pages, so we could just say something like

> Please refer to the system requirements documentation for your IDE to determine the adequate amount of resources for your workspace